src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31 calendar--start--day--mon">
+      <div class="calendar__day1 day"></div>
```

Comment:
```suggestion
      <div class="calendar_day"></div>
```

тебе не нужно давать каждому дню свой класс